Sources/Routing/Protocols/Dismissable.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  Created by Ahmed Fathy on 18/12/2023.
+//
+
+import SwiftUI
+
+public protocol Dismissable: ObservableObject {
```

Comment:
```suggestion
public protocol Dismissable {
```

I'm not sure this absolutely needs to be able to conform to `ObservableObject`